.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Unit Tests
```

Comment:
I'd rather name it CI, as we might have other jobs than pure tests. for instance, I tend to add a job running `composer validate` to catch mistakes in the composer.json

Reply:
Shall I add a `composer validate` step while I'm at it ?

Reply:
Went ahead and added the extra step.

Reply:
Just checking: would you like me to rename the file to `ci.yml` as well ?

Reply:
yes please

Reply:
Done

Reply:
Oops. Forgot to fix up the badge in the readme when I renamed the file. Fixed now.